core/cap-0035.md

```diff
@@ -268,7 +271,7 @@ index 7f08d757..ebbf45f5 100644
 +struct SetTrustLineFlagsOp
 +{
 +    AccountID trustor;
-+    AssetCode asset;
++    Asset asset;

```

Comment:
❗ The same change needs making to `ClawbackOp`. We agreed to also defer who can issue the `ClawbackOp` for the same reasons, that an issuer may be able to delegate that responsibility in the future at which point the operation must encode the issuer to reference the asset.

Reply:
Ah I missed this during the discussion but you're right. Fixed.